README.md

```diff
@@ -184,27 +157,28 @@ example of a config file content (see `./example.rainconfig.json`):
 ```
 
 ## **Developers**
-To get started, clone the repo and install the dependencies:
+## **Build From Source**
+To build from the source first clone this repo and make sure the following items are already installed:
```

Comment:
we use nix, all the instructions should be for nix

we don't support people BYO any of the below, we only support development with the shipped packages of these things in nix against the specific commits the flake is committed against for that supported code